lib/pascals_triangle.rb

```diff
@@ -0,0 +1,30 @@
+class PascalsTriangle
+
+  def initialize
+    @triangle = [[1]]
+  end
+
+  def one_row(row_num)
+    if @triangle[row_num]
+      return @triangle[row_num]
+    end
+    previous_row = one_row(row_num - 1)
+    @triangle << calculate(previous_row)
```

Comment:
@StefanGajic this won't work correctly if you don't call `one_row` in a sequence... actually it will work because it has to be in sequence because of `one_row` call on the line above.

Still, you can merge this and next line into one `@triangle[row_num] = calculate(previous_row)` it will have the same effect.